File: Chapter_2/xai/nursery.py
import os
from openai import OpenAI
from dotenv import load_dotenv

load_dotenv()

client = OpenAI(
    api_key=os.getenv("XAI_API_KEY"),
    base_url="https://api.x.ai/v1"
)

response = client.chat.completions.create(
    model="grok-3",
    messages=[{"role": "user", "content": "Twinkle, Twinkle, Little"}]
)

print(response.choices[0].message.content)

File: Chapter_2/xai/nursery.js
import OpenAI from "openai";
import "dotenv/config";

const xai = new OpenAI({
  apiKey: process.env.XAI_API_KEY,
  baseURL: "https://api.x.ai/v1",
});

const response = await xai.chat.completions.create({
  model: "grok-3",
  max_completion_tokens: 100,
  messages: [{ role: "user", content: "Twinkle, Twinkle, Little" }],
});

console.log(response.choices[0].message.content);
